### PR TITLE
PubSub: Update CPS Python library READMEs to indicate the library is now GA.

### LINKED ISSUE #8990: PubSub: Release the first GA version
If no major issues are discovered during the ongoing bake-in period, the PubSub client should be released as GA.

Releasing a new version is done with the help of [releasetool](https://github.com/googleapis/releasetool). Prior to the GA release, the following must be done:

- Go through the checklist from the internal GA release doc, make sure everything is in order.
- Changes in the client lib files:
    - Update PubSub Python lib README to indicate semver just like Java library has
    - Update beta tag in PubSub Python lib README
    - Update beta tag in Python libs root
    - Set the release level to `ga` in `.repo-metadata.json`
    - Change beta label in code samples in clients page in Google Cloud website
    - Add to release notes (talk to @kir-titievsky)
    - Update any documentation on Cloudsite

### PR DESCRIPTION
**Pull request by pradn**
…Also add some extra notes about versioning and contributions.

Closes #8990

### CODE REVIEW

File: pubsub/README.rst [hunk 1]
```diff
@@ -192,3 +192,25 @@ For example, to use JSON Web Tokens, provide a `google.auth.jwt.Credentials`_ in
...
+
+It is currently in major version one (``1.y.z``), which means that the public API should be considered stable.
+
+
+Contributing
+------------
+
+Contributions to this library are always welcome and highly encouraged.
+
+See `google-cloud`'s [CONTRIBUTING] documentation and the [shared documentation](https://github.com/googleapis/google-cloud-common/blob/master/contributing/readme.md#how-to-contribute-to-gcloud) for more information on how to get started.
```
Comment (plamut) [inline pubsub/README.rst]: The link target is missing for CONTRIBUTING

**Comment by busunkim96** [comment]
Could you also change the release level to `ga` in the .repo-metadata.json?

https://github.com/googleapis/google-cloud-python/blob/1fb3bc9de2500175f39fec4b126e3181d6c06006/pubsub/.repo-metadata.json#L7

**Comment by tseaver** [comment]
These changes should normally be part of the PR which makes the actual `google-cloud-pubsub 1.0.0` release (see #9140 for an example).

**Comment by plamut** [comment]
@busunkim96 Is that necessary in this PR, or is this done by the releasetool?

(here will be another PR following this one)

**Comment by busunkim96** [comment]
@plamut Releasetool doesn't do that automatically right now. I'll add a TODO to add that.

I agree with @tseaver. It'd be nicer for these changes to be made in the release PR.

**Comment by plamut** [comment]
@busunkim96 On it, will follow up with an autogenerated PR shortly.

**Comment by pradn** [comment]
Closing in favor of #9145

**Pull request closed without merging**